sys/mobileapps.yaml

```diff
@@ -4,39 +4,30 @@ paths:
   /v1/handling/rev/{route}/{title}:
     get:
       x-request-handler:
-        - get_rev:
-            request:
-              method: get
-              uri: /{domain}/sys/page_revisions/page/{title}
-              headers:
-                cache-control: '{cache-control}'
         - cache_branch:
```

Comment:
[meta comment, not specific to this patch]: I'm still having trouble understanding all this handler branching business. Do you think we could simplify this in a future iteration?